### ISSUE TITLE
No instructions to run GUI

### ISSUE THREAD

**Issue by ziplock9000** [EssexPAL/Meterman-Vc8145#1]
How do you run the GUI version?

**Comment by EssexPAL** [comment]
Hmm, I seem to have failed to upload the gui version or its instructions.  I will do this in the next 24 hrs.  My applogies.

**Comment by EssexPAL** [comment]
Hi ziplock9000.  I have uploaded the GUI version plus instructions and notes.  Please take a look at the notes because of the RS232 problems you may experience.

**Comment by ziplock9000** [comment]
Worked first time. Thank you very much.
Are you still actively developing this and looking for suggestions/bugs?

**Comment by EssexPAL** [comment]
Yes, I am happy to take suggestions and bug reports.  I am in the middle of two other projects right now but will get back to it a little later in the year.
